www/src/examples/Toast/PlacementMulti.js

```diff
@@ -0,0 +1,41 @@
+<div
+  aria-live="polite"
+  aria-atomic="true"
+  style={{
+    position: 'relative',
+    minHeight: '200px',
+  }}
+>
+  <div
+    style={{
+      position: 'absolute',
+      top: 0,
+      right: 0,
+    }}
+  >
+    <Toast>
+      <Toast.Header>
+        <img
```

Comment:
we've got a local version of placeholder: https://github.com/react-bootstrap/react-bootstrap/blob/master/www/src/components/PlaceholderImage.js that is in scope and a bit clearer to customize, e.g. no text, diff background color. See Card examples for usage